stream-video-android-ui-compose/src/main/kotlin/io/getstream/video/android/compose/theme/StreamColors.kt

```diff
@@ -17,229 +17,99 @@
-    public val disabled: Color,
-    public val borders: Color,
-    public val inputBackground: Color,
-    public val appBackground: Color,
-    public val barsBackground: Color,
-    public val linkBackground: Color,
-    public val overlay: Color,
-    public val overlayDark: Color,
-    public val primaryAccent: Color,
-    public val errorAccent: Color,
-    public val infoAccent: Color,
-    public val highlight: Color,
-    public val screenSharingBackground: Color,
-    public val screenSharingTooltipBackground: Color,
-    public val screenSharingTooltipContent: Color,
-    public val avatarInitials: Color,
-    public val activatedVolumeIndicator: Color,
-    public val deActivatedVolumeIndicator: Color,
-    public val connectionQualityBackground: Color,
-    public val connectionQualityBar: Color,
-    public val connectionQualityBarFilled: Color,
-    public val connectionQualityBarFilledPoor: Color,
-    public val participantLabelBackground: Color,
-    public val infoMenuOverlayColor: Color,
-    public val callFocusedBorder: Color,
-    public val callGradientStart: Color,
-    public val callGradientEnd: Color,
-    public val callDescription: Color,
-    public val callActionIconEnabledBackground: Color,
-    public val callActionIconDisabledBackground: Color,
-    public val callActionIconEnabled: Color,
-    public val callActionIconDisabled: Color,
-    public val callLobbyBackground: Color,
-    public val audioLeaveButton: Color,
-    public val audioActionColor: Color,
-    public val liveIndicator: Color,
-    public val audioIndicatorBackground: Color,
-    public val avatarBorderColor: Color,
-    public val participantContainerBackground: Color,
+    val brandPrimary: Color,
+    val brandPrimaryLt: Color,
+    val brandPrimaryDk: Color,
+    val brandSecondary: Color,
+    val brandSecondaryTransparent: Color,
+    val brandCyan: Color,
+    val brandGreen: Color,
+    val brandYellow: Color,
+    val brandRed: Color,
+    val brandRedLt: Color,
+    val brandRedDk: Color,
+    val brandMaroon: Color,
+    val brandViolet: Color,
+    val basePrimary: Color,
+    val baseSecondary: Color,
+    val baseTertiary: Color,
+    val baseQuaternary: Color,
+    val baseQuinary: Color,
+    val baseSenary: Color,
+    val baseSheetPrimary: Color,
+    val baseSheetSecondary: Color,
+    val baseSheetTertiary: Color,
+    val baseSheetQuarternary: Color,
+    val buttonPrimaryDefault: Color,
+    val buttonPrimaryPressed: Color,
+    val buttonPrimaryDisabled: Color,
+    val buttonBrandDefault: Color,
+    val buttonBrandPressed: Color,
+    val buttonBrandDisabled: Color,
+    val buttonAlertDefault: Color,
+    val buttonAlertPressed: Color,
+    val buttonAlertDisabled: Color,
+    val iconDefault: Color,
+    val iconPressed: Color,
+    val iconActive: Color,
+    val iconAlert: Color,
+    val iconDisabled: Color,
+    val alertSuccess: Color,
+    val alertCaution: Color,
+    val alertWarning: Color,
```

Comment:
This PR has already merged, but it would be good to have Kdocs for these properties for providing better developer experiences 😄